docs/setup.md

```diff
@@ -0,0 +1,259 @@
+
+```sh
+curl -X PUT -H 'X-Algolia-Application-Id: %YOUR_METIS_APPLICATION_ID' -H 'X-Algolia-API-Key: %YOUR_METIS_APPLICATION_API_KEY'  --data-binary '{
+  "extensions": {
+    "reranking": {
+      "endpoint": "%YOUR_RERANKING_ENDPOINT_URL",
+      "maxNbHits": 100,
+      "enabled": true
+    }
+  }
+ }'  "https://%YOUR_METIS_APPLICATION_URL/1/indexes/%YOUR_INDEX_NAME/settings"
+```
+
+### 3. Test your extension
+
+Test if your extension works as expected. The extension is enabled by default, so the search operation will directly return the reranked list of hits:
+
+```ts
+index.search('').then(({ hits }) => {
+  console.log(hits);
+});
+```
+
+Compare the returned result with a default result returned with deactivated extension:
+
+```ts
+index.search('', {
+  extensions: {
+    reranking: {
+      enabled: false,
+    }
+  }
+}).then(({ hits }) => {
+  console.log(hits);
+});
+```
+
+## Getting started reranking extension with Azure
+
+This guide requires to have a Docker image ready to deploy.
+The [source code for the example Docker image](/Docker) is available and deployed publicly on [Docker Hub](https://hub.docker.com/r/algolia/test-reranking).
+This image starts a webserver on `$PORT` (set through environment variables). It expects requests as described above, and returns `objectID`s in the reverse order they were sent.
+
+<br/>
+<br/>
+
+- Click `Create resource` button
+
+<img src="basics.jpg" alt="basics" align="right" width="500"/>
+
+- In the provided list of resources select "Web app"
+  - Select a resource group or create a new one if needed
+  - Input the app name
+  - Select `Docker container` on `Linux` operating system
+  - Choose the region
+    > :warning: Your Metis application and the extension must be hosted in the same region!
+  - Choose a service plan and click **Next**
+
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+
+<img src="docker.jpg" alt="docker" align="right" width="500"/>
+
+- In the Docker setup
+  - Select the `Single Container` option
+  - Select `Docker Hub` as the image source
+  - Select `Public` visibility
+  - Set `Image and tag` field with `algolia/test-reranking:1.0.2`
+  - Click the **Review + create** button. The validation of the app will take some time.
+
+<br/>
+<br/>
+<br/>
+
+- Once the validation passed, click `Create` button and wait until the deployment finished.
+- Open the created resource by clicking the **Go to resource** button.
+
+<br/>
+
+<img src="configuration.jpg" alt="configuration" align="right" width="500"/>
+
+- Select `Settings` -> `Configuration`
+- Click **New application setting**
+- Set name to `PORT` and value to `80`, click **OK** and then **Save**
+- You're all set. Go to the `Overview` section. You can find the URL of your application which can be used as the endpoint in the index settings.
+
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
```

Comment:
they were already there, but what is the br for?

Reply:
same comment

Reply:
This brs are to layout properly the images aligned to right (you can try to remove them in the editing mode and see the preview). If you know a more proper way to achieve this, don't hesitate to commit.